config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
+  get '/posts/', to: 'posts#index'
+  get '/posts/', to: 'posts#show'
+  get '/users', to: 'users#index'
+  get '/users/:id', to: 'users#show'
```

Comment:
### Optional
- [ ] It would be better if we remove all these lines of code since the resources routes takes care of that and more, we have made use of the nested routing mechanism where every post is associated with a user

Reply:
Thank you for the review. Done